src/plugins/cosmos/CosmosAssetAccountDetails.tsx

```diff
@@ -26,6 +28,7 @@ export const CosmosAssetAccountDetails = ({ assetId: caip19, accountId }: AssetD
           <AssetHeader assetId={caip19} accountId={accountId} />
           {accountId && <AccountAssets assetId={caip19} accountId={accountId} />}
           <AssetAccounts assetId={caip19} accountId={accountId} />
+          {FeatureFlag.CosmosInvestor && <StakingOpportunities />}
```

Comment:
I also like this and agree this is where it should live. If we're going to have plugins, there's no need to "pollute" the generic `<AssetAccountDetails />` with it.